turbopack/crates/turbopack-ecmascript-plugins/src/transform/emotion.rs

```diff
@@ -27,13 +28,21 @@ pub enum EmotionLabelKind {
 
 //[TODO]: need to support importmap, there are type mismatch between
 //next.config.js to swc's emotion options
-#[turbo_tasks::value(shared)]
+#[turbo_tasks::value(shared, eq = "manual")]
```

Comment:
Couldn't this just be
```suggestion
#[turbo_tasks::value(new)]
```
which then doesn't require Eq at all?

Reply:
Does this work? It seems like build fails

Reply:
Sorry, `cell = "new"`

Reply:
Nevermind, that that requires `PartialEq`